lib/rules/color-hex-alpha/__tests__/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: 'never',
+	accept: [
+		{
+			code: 'a{color:#888; }',
+		},
+		{
+			code: 'a{color:#888888; }',
+		},
+	],
+	reject: [
+		{
+			code: 'a{color:#8888;}',
+			message: messages.unexpected('#8888'),
+			line: 1,
+			column: 9,
+		},
+	],
+});
+testRule({
+	ruleName,
+	config: 'always',
+	accept: [
+		{
+			code: 'a { color: #8888; }',
+		},
+		{
+			code: 'a { color: #88888888; }',
+		},
+	],
+	reject: [
+		{
+			code: 'a{color:#888;}',
+			message: messages.expected('#888'),
+			line: 1,
+			column: 9,
+		},
+	],
+});
```

Comment:
Let's get an additional `testRule` block with `syntax: "scss" and an interpolation accept test case, e.g.

```css
a { color: #{ff}; }
```

```js
testRule({
	ruleName,
	config: ["never"],
	syntax: 'scss',

	accept: [
		{
			code: 'a { color: #{ff}; }',
			description: 'scss interpolation',
		},
	],
});